src/python/datapreprocessor/datapreprocessor/personclient.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+from ipersonclient import IPersonClient
+
+class PersonClient(IPersonClient):
+    def get_all(self):
+        response = self._get(self._build_url('people'))
+        items = response.json()['data']
+        return items
+
+
+    def get_by_name(self, person_name):
+        people = self.get_people()
```

Comment:
I feel like I'm missing something. Where are get_people and get_person defined? 🔍